fmriprep/cli/tests/test_parser.py

```diff
@@ -0,0 +1,116 @@
+"""Test parser."""
+from packaging.version import Version
+import pytest
+from ..parser import _build_parser
+from .. import version as _version
+from ... import config
+
+MIN_ARGS = ['data/', 'out/', 'participant']
+
+
+@pytest.mark.parametrize('args,code', [
+    ([], 2),
+    (MIN_ARGS, 2),  # bids_dir does not exist
+    (MIN_ARGS + ['--fs-license-file'], 2),
+    (MIN_ARGS + ['--fs-license-file', 'fslicense.txt'], 2),
+])
+def test_parser_errors(args, code):
+    """Check behavior of the parser."""
+    with pytest.raises(SystemExit) as error:
+        _build_parser().parse_args(args)
+
+    assert error.value.code == code
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/poldracklab/fmriprep/pullRequest?prid=5112966)